courses/koopa_troopa_beach/course_data.inc.c

```diff
@@ -8,13 +8,14 @@
 #include <code_80004740.h>
 
 #include <courses/koopa_troopa_beach/course_displaylists.inc.h>
+#include <courses/koopa_troopa_beach/course_textures.linkonly.h>
 
 Gfx d_course_koopa_troopa_beach_dl_0[] =
 {
     gsDPTileSync(),
     gsDPSetTile(G_IM_FMT_RGBA, G_IM_SIZ_16b, 8, 0x0000, G_TX_RENDERTILE, 0, G_TX_NOMIRROR | G_TX_WRAP, 5, G_TX_NOLOD, G_TX_NOMIRROR | G_TX_WRAP, 5, G_TX_NOLOD),
     gsDPSetTileSize(G_TX_RENDERTILE, 0, 0, 0x007C, 0x007C),
-    gsDPSetTextureImage(G_IM_FMT_RGBA, G_IM_SIZ_16b, 1, 0x05003800),
+    gsDPSetTextureImage(G_IM_FMT_RGBA, G_IM_SIZ_16b, 1, gKTBTexture660D8C),
```

Comment:
We should find a better naming convention for course textures.

Perhaps something like tex_ktb660D8C.

or maybe it makes sense to do some wacky abbreviations here like:
ktb_tex_660D8C.

I can merge as is for now if you don't want to go through this effort now.

However, `g` for global imo should only be used for actual game code and not for symbols that contain textures and the like.

Reply:
We can let like this but I will some rename of texture and model before merging it